src/DocumentFormat.OpenXml.Framework/Equality/OpenXmlElementEqualityOptions.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace DocumentFormat.OpenXml
+{
+    /// <summary>
+    /// Options defining the behaviour of equality for <see cref="OpenXmlElement"/>.
+    /// </summary>
+    public sealed class OpenXmlElementEqualityOptions
+    {
+        /// <summary>
+        /// Gets a value indicating whether extended attributes should be considered when determining equality.
+        /// </summary>
+        public bool IncludeExtendedAttributes { get; init; } = true;
+
+        /// <summary>
+        /// Gets a value indicating whether MC attributes should be considered when determining equality.
+        /// </summary>
+        public bool IncludeMCAttributes { get; init; } = true;
+
+        /// <summary>
+        /// Gets a value indicating whether namespace should alone be used when comparing idenity of elements, skipping prefix lookup.
+        /// </summary>
+        public bool CompareNamespaceInsteadOfPrefix { get; init; }
```

Comment:
I'm not sure I like the name of this... other ideas?

Reply:
Changed it to something else. If you also don't like that, please come up with one you like better :)